src/pages/console/uibuilder/slots.mdx

```diff
@@ -0,0 +1,150 @@
+export const meta = {
+  title: `Slots`,
+  description: `Add slots to Amplify Studio generated Figma-to-code components.  Used to support Nested Components or Collections in React code.`,
+};
+
+Slots allow you to nest other components as React code within your Studio UI components.  You can use slots to create dynamically generated children, like Comments on a Post, or to replace a child element altogether.
+
+## Adding a slot
+First, you'll need a component.  
+1. Log into Amplify and [navigate to Studio](/console/adminui/start/).
+
+2. On the left-hand navigation bar, click UI Library
+
+3. Select a Figma component from your UI Library that you've imported into Studio.  
+*If you don't have any Figma components already, you can start with [Amplify's Figma UI file](/console/uibuilder/figmatocode/).*
+
+4. Configure that component by clicking the Configure button in the upper right-hand corner.
+
+![Open your slot and click the gear icon to configure](/images/console/slots-console1.gif)
+
+Next, you'll add a slot to this component.  
+1. On the left-hand side, you'll see the Elements of your component.  Select a Figma Frame (![Figma frame icon](/images/console/figma-frame-icon.png)) within your component.  
+
+  In this example, the "Area" frame is selected.
+
+2. On the right-hand bar action bar, click the "Convert to a slot" button.  This will add a new component under the "Component properties" header.
+
+3. Optionally, change your your property name.  
+
+  In this example, the property has been renamed "comments"
+
+![Select your frame and add a slot](/images/console/slots-console2.gif)
+
+<Callout warning>
+
+Want to undo your slot creation?  
+
+Locate your slot in the Component properties, click the triple-dot icon, and click "Erase property" to remove the slot.
+
+</Callout>
+
+## Importing your component
```

Comment:
```suggestion
## Render the component with slots in React code
```

I don't like my suggestion either but something that points it clearer to the React component + slots would be good.